scipy/stats/_distn_infrastructure.py

```diff
@@ -2611,6 +2611,20 @@ def entropy(pk, qk=None, base=None):
     S : float
         The calculated entropy.
 
+    Examples
+    --------
+
+    >>> from scipy.stats import entropy
+
+    Bernoulli trial with different p:
+
+    >>> entropy([1/2, 1/2], base=2) # The outcome of a fair coin is the most uncertain
+    >>> entropy([9/10, 1/10], base=2) # The outcome of a biased coin is less uncertain
```

Comment:
our `refguide_check.py` machinery is correctly complaining that `entropy` returns a float, but no return value is provided in the docstring here